### ISSUE TITLE
arRPC shows as working but nothing is showing on user's profile

### ISSUE THREAD

**Issue by worf1337** [smartfrigde/armcord#536]
**Describe the bug**
Sorry if this isn't an ArmCord issue, I'm still getting used to everything.

Trying to get any rich presence doesn't work. In the logs (simply typing `armcord` in the terminal) arRPC seems to be working fine but nothing shows on the user's profile. 

**To Reproduce**
1. Open armcord
2. Open a program that uses RPC (e.g SuperTuxKart)
3. It shows as working in the logs but nothing shows on the profile

**Expected behavior**
It should show my rich status on my profile.

**Desktop (please complete the following information):**
 - OS: Linux
 - Method of installation: aur (armcord-git)
 - Version: none(?), installed by git

**Additional context**
I am using MATE. This didn't work on GNOME or i3 either though. It also works fine on Vencord Desktop.

```
[arRPC] arRPC v3.1.0 ArmCord
[arRPC > ipc] checking /run/user/1000/discord-ipc-0
[Config manager] skipSplash: false
[63734:1222/204703.902755:ERROR:object_proxy.cc(590)] Failed to call method: org.freedesktop.portal.Settings.Read: object_path= /org/freedesktop/portal/desktop: org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop
[arRPC > ipc] checked if socket is available: true 
[arRPC > ipc] listening at /run/user/1000/discord-ipc-0
[arRPC > websocket] trying port 6463
[arRPC > websocket] listening on 6463
[arRPC > process] started
[arRPC > ipc] new connection!
[arRPC > ipc] handshake: { v: 1, client_id: '817760324983324753' }
[arRPC > ipc] sending {
  cmd: 'DISPATCH',
  evt: 'READY',
  data: {
    v: 1,
    user: {
      id: '1045800378228281345',
      username: 'arRPC',
      discriminator: '0000',
      avatar: 'cfefa4d9839fb4bdf030f91c2a13e95c',
      flags: 0,
      premium_type: 0
    },
    config: {
      api_endpoint: '//discord.com/api',
      cdn_host: 'cdn.discordapp.com',
      environment: 'production'
    }
  }
}
[arRPC > ipc] message {
  cmd: 'SET_ACTIVITY',
  nonce: 1703238457,
  args: {
    activity: {
      state: 'Getting ready to race',
      assets: [Object],
      timestamps: [Object]
    },
    pid: 34069
  }
}
[arRPC > ipc] sending { cmd: 'SET_ACTIVITY', data: null, evt: null, nonce: 1703238457 }
[arRPC > ipc] message {
  cmd: 'SET_ACTIVITY',
  nonce: 1703238457,
  args: {
    activity: {
      state: 'Getting ready to race',
      assets: [Object],
      timestamps: [Object]
    },
    pid: 34069
  }
}
```

**Comment by ArjixWasTaken** [comment]
Related: #531

**Comment by ArjixWasTaken** [comment]
Can you test the latest release?

https://github.com/ArmCord/ArmCord/actions/runs/7346275092

**Comment by worf1337** [comment]
Seems to just have a blank screen on startup...

If it helps the console error logs are attached: [console-1704184034429.log](https://github.com/ArmCord/ArmCord/files/13808221/console-1704184034429.log)

Sorry if this is an issue on my end - I'm not experienced with Electron apps.

Sorry also for the late response time, I don't check GitHub frequently.

**Comment by h4rldev** [comment]
> Can you test the latest release?
> 
> https://github.com/ArmCord/ArmCord/actions/runs/7346275092

Haha, sorry for barging, just saying hi

**Comment by s-liwka** [comment]
same issue, i got armcord from chaotic-aur which seems to have commit #531 and it still doesnt work

**Comment by smartfrigde** [comment]
Fixed in 3.2.6
